tools/travis/script.sh

```diff
@@ -10,6 +10,11 @@ if [[ $TRAVIS_OS_NAME == "osx" ]]; then
     echo 'backend : Template' > $MPL_DIR/matplotlibrc
 fi
 
+section "list.installed.dependencies"
+pip list
+tools/build_versions.py
```

Comment:
Same here.

Reply:
QT isn't included in the requirements file, so to see what was installed, I had to use the `pip list` command.

Reply:
Maybe it would be clearer if this was moved to the `install` section?